src/index.html

```diff
@@ -1,15 +1,145 @@
+          Full date of birth:
+          <input type="date" id="user-birthday">
+        </label>
+        <label for="agreement">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+            id="agreement">
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <label for="user-email">
+          E-mail:
+          <input
+            type="email"
+            name="e-mail"
+            placeholder="email@example.com"
+            id="user-email"
+            required multiple>
+        </label>
+        <label for="user-password">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="6"
+            maxlength="20"
+            id="user-password">
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <label for="yes">
+          Do you love cats?
+          <input
+            type="radio"
+            name="loving-cats"
+            id="yes">
+          <label for="yes">Yes</label>
+          <input
+            type="radio"
+            name="loving-cats"
+            id="no">
+          <label for="no">No</label>
+        </label>
+        <label for="user-favorite-color">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            id="user-favorite-color">
+        </label>
+        <label for="user-go-to-bed-time">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="go-to-bed-time"
+            id="user-go-to-bed-time">
+        </label>
+        <label for="user-favorite-brand-car">
+          What are your favorite brand of car?
+          <select
+            name="brand-of-car"
+            id="user-favorite-brand-car"
+            multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+        <label for="estimate-our-work">
+          How do you rate our work?
+          <input
+            type="range"
+            value="0"
+            id="estimate-our-work">
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Additional info:</legend>
+        <label for="user-comments">
+          Comments:
+          <textarea
+            name="comments"
+            cols="20"
+            rows="2"
+            id="user-comments"
+         ></textarea>
+        </label>
+        <label for="user-recommendation">
+          Would you recommend us?
+          <select name="recommendation" id="user-recommendation">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+      <button name="submit" type="submit">Submit</button>
```

Comment:
no need for name, type="submit" is e default type for a button

```suggestion
      <button>Submit</button>
```